.github/workflows/openapi.yml

```diff
@@ -0,0 +1,38 @@
+name: API Quality
```

Comment:
Perhaps just `OpenAPI spec validation` or something similar would be clearer?

Reply:
I was inspired by "Coding Standards" of the other actions :-) . But in reality we have only one check here .. changing it